### PR TITLE
test: add tests for peribolos as a service installation

### PR DESCRIPTION
**Pull request by KPostOffice**
closes: #37

### CODE REVIEW

File: src/probotOnKube.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Probot } from 'probot';
 import fs from 'fs';
-import k8s, { V1Secret } from '@kubernetes/client-node';
+import { V1Secret } from '@kubernetes/client-node';
+import * as k8s from '@kubernetes/client-node';
```
Comment (harshad16) [inline src/probotOnKube.ts:4]: why this change? does it provide some different functionality?

**Review by harshad16 — COMMENTED** [review]
lgtm
great pr :100: 
just one question ?

File: src/probotOnKube.ts (hunk 1 above)
Comment (KPostOffice) [reply src/probotOnKube.ts:4]: '@kubernetes/client-node' didn't provide any default exports from what I could tell. My linter was giving warnings.

Comment (harshad16) [reply src/probotOnKube.ts:4]: ack, all good to me.
you would have to add signed commit to pass the DCO check.
use the `-s` argument while committing.

**Comment by Gregory-Pereira** [comment]
ping @KPostOffice, is there an update on this? Trying to close out old issues and this looks good just failing some checks.

**Comment by KPostOffice** [comment]
@Gregory-Pereira Hey. I need to work through this with someone. I had it working, but then the code was rewritten to abstract away the k8 specific code into a separate package which ended up breaking the test I wrote. Now I'm having a lot of trouble mocking out the tests.
